src/huggingface_hub/inference/_client.py

```diff
@@ -334,9 +323,12 @@ def audio_classification(
         audio: ContentT,
         *,
         model: Optional[str] = None,
+        function_to_apply: Optional[Literal["sigmoid", "softmax", "none"]] = None,
+        top_k: Optional[int] = None,
     ) -> List[AudioClassificationOutputElement]:
         """
         Perform audio classification on the provided audio content.
+        For more details about the input parameters, see the [pipeline documentation](https://huggingface.co/docs/transformers/en/main_classes/pipelines#transformers.AudioClassificationPipeline).
```

Comment:
I find it a bit confusing to redirect from Inference docs to the `transformers` pipeline docs. I feel that we should better document the parameters if that's what's missing here.

Because here for example https://huggingface.co/docs/transformers/en/main_classes/pipelines#transformers.AudioClassificationPipeline.__call__ do not provide any information about `function_to_apply`, meaning we would have to maintain both docs. Better not to have this additional "docs dependency".

Reply:
I added this because for some fields, the default values are only documented in the pipeline docs, e.g. for `text-classification` [here](https://huggingface.co/docs/transformers/en/main_classes/pipelines#transformers.TextClassificationPipeline.__call__)
but you're right, it's still an additional doc dependency but it would be nice to have access to the default values in the inference doc.

Reply:
Yes indeed, we should better document the specs themselves but it will be done in separate PRs :confused: